packages/react/src/components/Authenticator/VerifyUser/VerifyUser.tsx

```diff
@@ -17,10 +18,9 @@ const generateRadioGroup = (
 
   for (const [key, value] of Object.entries(attributes)) {
     const radio = (
-      <label key={key}>
-        <input type="radio" name="unverifiedAttr" value={key} />
+      <Radio name="unverifiedAttr" value={key} key={key}>
```

Comment:
If there's a single attribute only (Email), would it make sense to use a checkbox instead? Seems odd to show a radio button for a single option as shown in the screenshot. IME, radio buttons are usually used for picking between at least two options.

Reply:
There can be multiple options. It's just that the example I'm using only has `email` as an attribute to verify.